roles/taca/templates/site_taca_delivery.yml.j2

```diff
@@ -0,0 +1,85 @@
+log:
+    file: "{{ ngi_pipeline_{{site}}_path }}/log/taca.log"
```

Comment:
I have never tried doing nested variable expansions like this. Do you know if this will work? 

Otherwise use a generic variable name in the template that you set to a site-specific value in the `set_fact` before the `template` task.